src/BlissSymbol.test.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2023 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+import { render, screen } from "@testing-library/preact";
+import "@testing-library/jest-dom";
+import { html } from "htm/preact";
+
+import { initAdaptivePaletteGlobals } from "./GlobalData";
+import { BlissSymbol } from "./BlissSymbol";
+
+test("The BlissSymbol component is rendered correctly", async () => {
+
+  await initAdaptivePaletteGlobals();
+
+  const singleBciAvId = {
+    bciAvId: 12335,
+    label: "VERB"
+  };
+
+  const arrayBciAvId = {
+    bciAvId: [12335, "/", 8499],
+    label: "VERB+S"
+  };
```

Comment:
It might be clearer to split this test into two sections or two tests, one for testing the number bciAvId and the other for testing the array bciAvId.

Reply:
Okay, done.